common/delay.c

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <delay.h>
+#include <stdint.h>
+#include <drivers/wallclock.h>
+
+void
+delay_cycles(uint32_t cycles)
```

Comment:
If this is deemed necessary, this is something that would be _included_ with the timer driver. And these changes are not merged, I am unsure why they are present in both #76 and this PR.

Reply:
Because both the timer and clock PRs are currently based on top of #76, since they require this function.